openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -142,10 +145,11 @@ def test_calculate_course_xblocks_data(self, course_attr):
                     )
 
     @ddt.data(
-        ('course', 47),
-        ('other_course', 34)
+        ('course', 48),
+        ('other_course', 35)
     )
     @ddt.unpack
+    @skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Different query counts in CMS')
```

Comment:
The query count did not change when running from CMS. The vast majority of bookmarks tests are skipped in CMS anyway, so skipping another doesn't seem too bad.